content/markdown/01-basic-r/01-rmarkdown-examples.rmd

```diff
@@ -0,0 +1,107 @@
+At the most basic level, we can use R as a (very complicated) calculator. We can perform all the most basic operations that we might be interested in:
```

Comment:
I'm a little confused about this part. This is the text from the coding basics section? Not sure what I'm looking at in the examples.